Cabal/Distribution/PackageDescription/Parse.hs

```diff
@@ -161,6 +162,14 @@ storeXFieldsPD _ _ = Nothing
 -- ---------------------------------------------------------------------------
 -- The Library type
 
+subLibFieldDescrs :: [FieldDescr Library]
+subLibFieldDescrs =
+  [ simpleField "library"
+                        showToken             parseTokenQ
+                        (fromMaybe "subLibFieldDescrs" . libName)
```

Comment:
It's supposed to be `error "subLibFieldDescrs"`. Invariant is you call these where it's `Just`.